### ISSUE TITLE
hPa to mmHG conversion and localization features request

### ISSUE THREAD

**Issue by cepxuo** [bramkragten/weather-card#22]
Hi! Thanks for amazing module!
Is it possible to include couple new features? I would like to have an option to display pressure in mmHg. Also it would be great to have localization option. I can help with ru_RU translation.
# #

**Comment by ElwayDenmark** [comment]
And I would like to have wind speed in m/s instead of km/h. The m/s is the default in Denmark.

**Comment by TEDSv** [comment]
Same question :)

**Comment by Dcashe** [comment]
+1 Same question

**Comment by TTLucian** [comment]
Yeah. Card is showing 1006 hpa instead of 1006 mmHg (which is sensor's default unit of measurement}.

**Comment by Nikolay-Ch** [comment]
+1 and how can I change language ot the card insteed use system local?
